app/src/main/java/com/islaharper/dawnofandroid/domain/repository/AuthRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.islaharper.dawnofandroid.domain.repository
+
+import com.google.android.gms.auth.api.identity.BeginSignInResult
+import com.islaharper.dawnofandroid.util.Resource
+
+typealias OneTapSignInResponse = Resource<BeginSignInResult>
```

Comment:
It's strictly not necessary, my thinking is that with Resource able to be used in a few different ways, it might be useful to differentiate them? If you're happy with this approach I can refactor to use descriptive aliases in the other places Resource is use, else I can remove it.

Reply:
My personal preference would be to remove the type alias unless we can prove it provides a significant benefit. Using `Resource<>` with a type parameter is clearer to me as a reader.

Reply:
Refactored and PR updated